docs/pages/modules/supporting-platforms.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Supporting more platforms
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
+
+<Step label="1">
+
+### Use the `"apple"` platform in `expo-module.config.json`
+
+In order to provide more seamless support for other Apple platforms, Expo SDK 50 introduced a universal `"apple"` platform to instruct the autolinking that the module may support any of the Apple platform
```

Comment:
```suggestion
To provide seamless support for other Apple platforms, Expo SDK 50 introduced a universal `"apple"` platform to instruct the autolinking that the module may support any of the Apple platform
```

Let's avoid being too wordy when we can use a single word for the same meaning. Vale also warns and suggests about this:

![CleanShot 2024-02-01 at 15 57 47@2x](https://github.com/expo/expo/assets/10234615/835e9f7d-547f-4631-a6b3-f9f7ca6a8d52)